src/PrimeBooking.Domain/PrimeBooking.Domain.csproj

```diff
@@ -6,4 +6,12 @@
         <Nullable>enable</Nullable>
     </PropertyGroup>
 
+    <ItemGroup>
+      <Folder Include="Structural\" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Include="MediatR" Version="12.4.0" />
```

Comment:
As its pet project we can think of changing MediatR with Source Generated MediatR. Only for interest: https://www.youtube.com/watch?v=aaFLtcf8cO4

Reply:
Nice catch, uploaded new package